redfish-core/lib/fabric_adapters.hpp

```diff
@@ -199,6 +199,77 @@ inline void linkAsPCIeDevice(const std::shared_ptr<bmcweb::AsyncResp>& aResp,
         deviceArray.size();
 }
 
+inline void getFabricAdapterPCIeSlots(
+    const std::shared_ptr<bmcweb::AsyncResp>& aResp,
+    const std::string& systemName, const std::string& fabricAdapterPath,
+    std::function<void(const std::string& /*chassisName*/,
+                       const dbus::utility::MapperEndPoints&)>&& callback)
+{
```

Comment:
Are we planning to upstream this code? The lambda function is too long and does not adhere to the coding guidelines.

Reply:
As this time, this is downstream only, as this association is a temporary workaround for 1050 or earlier releases.
That's why it is under OEM schema.